### PR TITLE
Add option to make new window frameless

### PR DESCRIPTION
**Pull request by BenLambertNcl**
Only works in Electron and OpenFin. Also added option to make window transparent.

### CODE REVIEW

File: packages/api-electron/index.js [hunk 1]
```diff
@@ -25,13 +25,14 @@ module.exports = (appJson) => {
...
-        sandbox: true,
-        preload: preloadPath
-      }
+    const webPreferences = {
+      sandbox: true,
+      preload: preloadPath
     };
 
+    const options = msg.features;
+    options.webPreferences = webPreferences;
```
Comment (AndyMJones) [inline packages/api-electron/index.js]: I'd probably use `Object.assign` here instead
```
const options = Object.assign(
  {}, 
  message.features, 
  { webPreferences: { sandbox: true, preload: preloadPath } }
);

File: packages/api-openfin/src/window.js [hunk 2]
```diff
@@ -10,23 +10,29 @@ class Window {
...
-      if (features && features.child) {
-        newWindow = new fin.desktop.Window({
+      const windowOptions = features || {};
+      if (windowOptions.transparent) {
+        // OpenFin needs opacity between 1 (not transparent) and 0 (fully transparent)
+        windowOptions.opacity = features.transparent === true ? 0 : 1;
+      }
+
+      if (windowOptions.child) {
+        Object.assign(windowOptions, {
```
Comment (AndyMJones) [inline packages/api-openfin/src/window.js]: I'm not a fan of overriding `windowOptions`. I'd assign a new `const` and use that instead

**Comment by BenLambertNcl** [comment]
Done :)

**Review by AndyMJones — APPROVED** [review]
lgtm

**Pull request merged**